cloudnew/ec2_util.go

```diff
@@ -0,0 +1,142 @@
+package cloudnew
+
+import (
+	"math"
+	"os"
+	"os/user"
+	"strings"
+	"time"
+
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/evergreen-ci/evergreen/cloud"
+	"github.com/evergreen-ci/evergreen/model/host"
+	"github.com/pkg/errors"
+)
+
+const (
+	HostExpireDays      = 10
+	NameTimeFormat      = "20060102150405"
+	SpawnHostExpireDays = 30
```

Comment:
why public? should we move these to evergreen/globals.go?

Reply:
done